README.md

```diff
@@ -19,97 +20,161 @@ This WordPress project is intended to provide:
 - Homebrew (1.2.3)
   - MariaDB (10.0 or greater)
   - Wordpress Command Line (1.2.1)
+  - Composer (1.4.2)
 - PHP 7
+- Codeception for Wordpress (Wp-Browser 1.21)
+
+***
 
 ## Installation
 
-  Installation process for the setting up Wordpress on a local environment.
+Installation process for the setting up Wordpress on a local environment.
+
+- **Create the MySQL user and databases using the MariaDB monitor**.
+
+```
+mysql -uroot
+CREATE DATABASE chappress_dev;
+GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+
+CREATE DATABASE chappress_test;
+GRANT ALL PRIVILEGES ON chappress_test.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+
+CREATE DATABASE chappress_test_wpunit;
+GRANT ALL PRIVILEGES ON chappress_test_wpunit.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+
+FLUSH PRIVILEGES;
+EXIT
+```
 
-- Create the MySQL user and database using the MariaDB monitor.
+- **Download Chap-Press Git with WordPress**
 
-  ```
-  mysql -uroot
-  CREATE DATABASE chappress_dev;
-  GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
-  FLUSH PRIVILEGES;
-  EXIT
-  ```
+```
+git clone git@github.com:chapmanu/chap-press.git
+cd chap-press/public
+cp -v wp-config.php{-dist,}
 
-- Download WordPress and run the server.
+# The wp-config file already contains the database information to get started.
+# These are for local development only.
+```
 
-  ```
-  git clone git@github.com:chapmanu/chap-press.git
-  cd chap-press/public
-  cp -v wp-config.php{-dist,}
+- **Install Automated Test Suite**
 
-  # The wp-config file already contains the database information to get started.
-  # These are for local development only.
+Navigate to project folder root: `/chap-press`
 
-  php -S localhost:8000
-  ```
+```
+composer install
+#installs depedency from composer.json
 
-  Server should be running in the terminal displaying PHP version, port location, and document root.
+echo "alias codecept=./vendor/bin/codecept" >> ~/.bash_profile
+#creates an alias
 
-  Go to your browser and enter the url port:
-  ```
-  http://localhost:8000
-  ```
+source ~/.bash_profile
 
-  Finish the final instructions through the WordPress installation. The WordPress admin panel should be displayed and the user should have full access to developing in a local environment.
+codecept --version
+```
+
+Codeception executed as `codecept` or `./vendor/bin/codecept`
+
+- **Run the Server**
+
+**Execute**: `./run-server.sh`
```

Comment:
Reiterating comment on earlier changeset: remove this script and just list the php command here.